dataflow/manual/content.tex

```diff
@@ -1782,6 +1782,31 @@ \section{Example: Very Busy Expression Analysis}
 
 \flow{BusyExprSimple}{.33}{1}{Very Busy Expression Analysis illustration. Intermediate analysis results are shown.}
 
+\section{Example: Reaching Definitions}
+
+The reaching definitions for a given program point are those assignments that
+may have updated the current values of variables.
+This reaching definitions analysis is a standard example of a forward analysis.
+
+\textbf{The node.} A class \code{ReachingDefinitionsNode} is used as an
```

Comment:
Should these be renamed to `Abstract values`/`Value` like in BusyExpr above?

Reply:
Thanks, done in my local.